sources/packages/backend/libs/integrations/src/esdc-integration/msfaa-integration/msfaa-files/msfaa-file-detail.ts

```diff
@@ -54,7 +54,7 @@ export class MSFAAFileDetail implements MSFAARequestFileLine {
     record.appendWithEndFiller(this.provinceState ?? "", 4, SPACE_FILLER);
     record.appendWithEndFiller(this.postalCode ?? "", 16, SPACE_FILLER);
     record.appendWithEndFiller(this.country, 20, SPACE_FILLER);
-    record.appendWithEndFiller(this.phone ?? "", 20, SPACE_FILLER);
+    record.appendWithStartFiller(this.phone ?? "", 20, NUMBER_FILLER);
```

Comment:
`phone` can be declared as a `number`.

Reply:
`NUMBER_FILLER` is a string. So, wouldn't it be better to keep it as a string. Plus, the problem of number overflowing (number > `Number.MAX_VALUE`) won't be there if we keep it as string. Am I missing something here?

Reply:
About `NUMBER_FILLER` being a string it does not justify at all why the `phone` cannot be a number.
About the second part, the max safe integer would be something around `9007199254740991` which is pretty big for a phone number. Since it is a JSONB and it would be allowed in DB, for this reason and only for this reason I am ok to keep it as string.